delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/resources/ResourcesImpl.scala

```diff
@@ -216,13 +216,7 @@ object ResourcesImpl {
       initialState = Initial,
       next = Resources.next,
       evaluate = Resources.evaluate(resourceResolution),
-      tagger = (ev: ResourceEvent) =>
-        Set(
-          Event.eventTag,
-          moduleType,
-          Projects.projectTag(ev.project),
-          Organizations.orgTag(ev.project.organization)
-        ),
+      tagger = EventTags.forProjectScopedEvent(moduleType),
```

Comment:
I understand that for light and rare events, we might want to add more tags but for resources, the cost may be a little high to have 6 tags for resources no ?

Reply:
We agreed when we discussed that issue that we are going to do it for all events.

Reply:
It is precisely for resources and files that this is going to become useful.